AlphaWallet/Resources/Assets.xcassets/wallet_security_green_bar.imageset/Contents.json

```diff
@@ -1,5 +1,33 @@
 {
   "images" : [
+    {
```

Comment:
The rest looks good, but looks like this shouldn't change?

Reply:
don't know, maybe it because i deleted images related to artis networks?

Reply:
Maybe, but seems odd that Xcode modifies an unrelated image. Think we can just remove them.